### ISSUE TITLE
Trump administration asks Supreme Court to stop release of inmates at risk for COVID-19

### ISSUE THREAD

**Issue by cronanews** [cronanews/VirusUpdate#190]
<b>Trump administration asks Supreme Court to stop release of inmates at risk for COVID-19</b><br>
<p>
 <ins class="adsbygoogle" style="display:block;" data-ad-client="ca-pub-4502326097154138" data-ad-slot="3192338349" data-ad-format="auto" data-full-width-responsive="true"></ins> 
<br>
The Supreme Court has been asked to stop the release of federal prison inmates vulnerable to the coronavirus in another emergency appeal from the Trump administration. (Jacquelyn Martin / Associated Press)</p>
<p>The Trump administration is asking the Supreme Court to intervene for the first time in a case related to COVID-19, to block a judge&rsquo;s order that a low-security federal prison in Ohio transfer, release or send home some of its elderly and medically vulnerable inmates.</p>
<p>&ldquo;The government is currently facing numerous suits challenging conditions of confinement in federal prisons across the nation,&rdquo; Solicitor Gen. Noel Francisco said in an appeal filed Friday evening, and the prisoners&rsquo; lawyers are seeking to require the wholesale release of inmates from low-security prisons by imposing &ldquo;a constitutional six-feet-at-all-times rule&rdquo; for social distancing.</p>
<p>The court, which could act this weekend, is thrust into the fray as state and federal prisons nationwide have become hot spots for infections and illness from the coronavirus, and prison officials have searched for ways to thin their populations. Earlier Friday, lawyers for the American Civil Liberties Union told the justices it would be &ldquo;a tremendous mistake&rdquo; for the high court to intervene. They noted the Ohio judge had yet to order the release of a single inmate.</p>
<p>Granting the appeal would &ldquo;block an order that will save lives,&rdquo; said David Cole, the ACLU&rsquo;s national legal director. &ldquo;Around the country, courts have been slow to step in and take responsibility to protect the tens of thousands of incarcerated people who are at risk from this virus. Bold leadership from courts will be required to mitigate the humanitarian crisis we&rsquo;re facing.&rdquo;</p>
<p>The Elkton Federal Correctional Institution in northeast Ohio has been hard hit by the coronavirus. By early May, nine inmates had died, and one in four of those tested were positive.</p>
<p>A lawsuit filed on behalf of four inmates alleged that the conditions there were inhumane and unconstitutional, with 2,500 in</p>
<p>mates housed in cramped dormitory-style facilities, and crowded together when they slept, ate or bathed.</p>
<p>In response, U.S. District Judge James S. Gwin on April 22 ordered the prison to evaluate its prison population and identify those who were at high risk because they were 65 or older or had medical conditions like asthma or a heart ailment that put them in danger if they contracted the virus.</p>
<p>Story continues</p>
<p>The prison subsequently said 837 inmates were in those categories. But officials later said only five of the inmates were suitable candidates to be confined at home or released. The lawyers who brought the suit expressed surprise, given that Atty. Gen. William Barr had told the federal Bureau of Prisons in late March to consider early release or home confinement in response to the virus.</p>
<p>Frustrated, the judge issued a new order on May 19 telling prison officials to send home or release the high-risk inmates or transfer them to another prison, or explain why they did not do so in each case. He said he wanted all the inmates to be reevaluated by May 26.</p>
<p>On Wednesday , the solicitor general went directly to the Supreme Court. Francisco said the justices should stay the judge&rsquo;s order, blocking it while the government appeals to the Cincinnati-based 6th Circuit Court of Appeals, or, if that fails, seeks a full review before the high court. The case is Williams vs. Wilson.</p>
<p>In past administrations, the government rarely sent emergency appeals to the high court. Typically, the solicitor general seeks review of a lower court ruling once the case has been finally decided by an appeals court. But since President Trump took office, the pattern has shifted. As his actions have been quickly challenged in federal courts, and often blocked by judges, just as regularly Francisco has gone directly to the high court and asked the justices to lift or set aside a lower court order.</p>
<p>Francisco has been successful most of the time. On Wednesday, for example, the justices issued a brief order to temporarily block House Democrats from seeing files and interviews from the investigation of special counsel Robert S. Mueller III into Russian meddling in the 2016 election. Two lower courts ruled the material must be turned over to the House, but the justices agreed to put those rulings on hold while Francisco submits a full appeal.</p>
<p>In the Ohio prison case, Francisco argued that the judge had overstepped his authority and evaded the Prison Litigation Reform Act of 1995, which bars inmates from challenging prison conditions in federal court until they have tried to obtain relief through the prison grievance system.</p>
<p>Gwin had found that Elkton&rsquo;s inhumane conditions violated the 8th Amendment banning cruel and unusual punishment, and said the need to remedy this constitutional violation justified his intervention.</p>
<p>Francisco called the action &ldquo;judicial second guessing.&rdquo; He said Gwin&rsquo;s order &ldquo;fails to account for the practical constraints facing prison administrators managing the nation&rsquo;s prison system during a public-health emergency, and ignores the actual and extensive steps that they have taken to protect inmates from the risk of infection within those constraints.&rdquo;</p>
<p>The ACLU lawyers said the pandemic requires an emergency response like the one Gwin ordered. &ldquo;COVID-19 poses a particularly acute danger at Elkton, where prisoners are forced to live in a single dormitory room along with approximately 150 other people,&rdquo; they told the court. &ldquo;The government has limited the ability of [the inmates] to maintain adequate distance from each other, requiring them to sleep, eat, and live, just a few feet from other potentially contagious prisoners.&rdquo;</p>
<p>The justices also have before them a similar, but more narrow appeal from a New Orleans lawyer who is seeking relief for a Louisiana inmate who has diabetes. In that case, the 5th Circuit Court of Appeals blocked a federal district judge&rsquo;s order in favor of the inmate.</p>
<p>Two weeks ago, the high court turned down an emergency appeal from two elderly inmates in a Texas state prison who sought more protections from COVID-19. The 5th Circuit also had blocked a judge&rsquo;s order for those inmates, citing their failure to file an appeal or grievance with the prison system.</p>
<p>Justice Sonia Sotomayor agreed with the narrow ruling in the Texas case, Valentine vs. Collier, but wrote in a statement joined by Justice Ruth Bader Ginsburg that judges needed to do more to protect inmates during this emergency.</p>
<p>&ldquo;It has long been said that a society&rsquo;s worth can be judged by taking stock of its prisons. That is all the truer in this pandemic, where inmates everywhere have been rendered vulnerable and often powerless to protect themselves from harm,&rdquo; she said. &ldquo;May we hope that our country&rsquo;s facilities serve as models rather than cautionary tales.&rdquo;</p>
<p>
 <ins class="adsbygoogle" style="display:block;" data-ad-client="ca-pub-4502326097154138" data-ad-slot="3192338349" data-ad-format="auto" data-full-width-responsive="true"></ins> 
 
 
 
 
<br>
<br>
<a href="https://news.yahoo.com/trump-administration-asks-supreme-court-193138543.html" target="_blank" rel="noopener noreferrer">Source link</a></p>
<br>
<br>
from WordPress <a href="https://www.coronavirus-update.me/2020/05/24/trump-administration-asks-supreme-court-to-stop-release-of-inmates-at-risk-for-covid-19/">https://www.coronavirus-update.me/2020/05/24/trump-administration-asks-supreme-court-to-stop-release-of-inmates-at-risk-for-covid-19/</a><br>
<br>
via Blogger https://ift.tt/36vbumX<br>
May 24, 2020 at 08:04PM
